notebooks/quickstart.ipynb

```diff
@@ -120,18 +122,115 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "--------------------------------\n",
+    "## The basics of `spaghetti`\n",
+    "### 1. Creating a network instance\n",
+    "\n",
+    "Within `spaghetti` network objects can be created from [a variety of objects](https://pysal.org/spaghetti/generated/spaghetti.Network.html#spaghetti-network), the most common being shapefiles (read in as file paths) and `geopandas.GeoDataFrame` objects. However, a network could also be created from `libpysal` geometries, as demonstrated in the [connected components tutorial](https://pysal.org/spaghetti/notebooks/connected-components.html#1.-Instantiate-a-network-from-two-collections-of-libpysal.cg.Chain-objects) or a simply as follows:\n",
+    "\n",
+    "```python\n",
+    "from libpysal.cg import Point, Chain\n",
+    "import spaghetti\n",
+    "# create the network\n",
+    "ntw = spaghetti.Network(in_data=Chain([Point([1, 1]), Point([2, 1])]))\n",
+    "```\n",
+    "\n",
+    "This will create a single-segment network.\n",
+    "\n",
+    "The stucture and characterstics of the networks are topics of research in many areas. However, networks are also utilized as the study space in many applications. In these cases the actual objects of interest are network-based events.\n",
+    "\n",
+    "\n",
+    "### 2. Snapping events (points) to a network\n",
+    "\n",
+    "\n",
+    "Points must be snapped to the network for any meaningful models to be constructed. As with `spaghetti.Network` objects, `spaghetti.PointPatten` more often [created from](https://pysal.org/spaghetti/generated/spaghetti.PointPattern.html#spaghetti.PointPattern) shapefile and `geopandas.GeoDataFrame` objects, but can also simply be a single `libpysal.cg.Point` object. Considering the single-segment network above:\n",
```

Comment:
"""
First, point objects, representing our network-based events, must be snapped to the network for meaningful spatial analysis to be done or models to be constructed. As with `spaghetti.Network` objects, `spaghetti.PointPatten` can be created from shapefiles and geopandas.GeoDataFrame objects. Furthermore, `spaghetti` can also simply handle a single libpysal.cg.Point object. Considering the single-segment network above:

"""